overlord/snapstate/backend_test.go

```diff
@@ -238,6 +239,8 @@ func (f *fakeStore) snap(spec snapSpec) (*snap.Info, error) {
 		confinement = "classic"
 	case "some-epoch-snap":
 		epoch = snap.E("42")
+	case "snapd-desktop-integration":
+		snapID = "IrwRHakqtzhFRHJOOPxKVPU0Kk7Erhcu"
```

Comment:
Small nit: We could define `const SnapdDesktopIntegrationSnapID = snapdDesktopIntegrationSnapID` in `overlord/snapstate/export_test.go` and then use that const here. That way we wouldn't need to redefine it here and it would be easier to match it to the relevant logic in `snapstate.go`

Reply:
Fair enough :+1: thanks for explaining the rationale